source/AI.cpp

```diff
@@ -4099,7 +4102,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 				message = "Landing on " + target->Name() + ".";
 		}
 		if(!message.empty())
-			Messages::Add(message, Messages::Importance::High);
+			Messages::Add(message, messageImportance);
```

Comment:
I agree with @Amazinite, using the highest importance seems appropriate and should alleviate any issues with the message blending in with others.
```suggestion
			Messages::Add(message, Messages::Importance::Highest);
```

Reply:
If I will apply this change every message will be change in this scope. I've added separate variable specificly to be able to modify one message. In case you will need to change Importance you'll just set it to messageImportance variable.

Reply:
Sorry, I think I've misread part of this (probably also because it's changed as I was reading an earlier version.)
Don't make this change.
As @Amazinite suggests, I think the other case where you can't land because the government doesn't like you should also be "Highest" importance.

Reply:
Done :)